packages/block-library/src/table-of-contents/edit.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Internal dependencies
+ */
+import * as Utils from './utils';
+import ListLevel from './ListLevel';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { subscribe } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import {
+	IconButton,
+	Toolbar,
+	PanelBody,
+	ToggleControl,
+} from '@wordpress/components';
+import { BlockControls, InspectorControls } from '@wordpress/editor';
+
+class TOCEdit extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			wpDataUnsubscribe: null,
+		};
+
+		this.toggleAttribute = this.toggleAttribute.bind( this );
+		this.refresh = this.refresh.bind( this );
+	}
+
+	toggleAttribute( propName ) {
```

Comment:
This method is used in a single place. Is it useful instead of simply setting the attribute to ! attribute?